example-workflows/connector/.github/workflows/docs.yml

```diff
@@ -9,6 +9,12 @@ on:
     paths:
       - "templates/docs/*"
 
+env:
+  API_TOKEN_GITHUB: ${{ secrets.CI_TOKEN }}
+  CONNECTOR_NAME: 'CHANGE ME' # TODO: change the connector name
```

Comment:
why is the connector-name necessary here? Should be derived by the gradle properties, else this is a duplication

Reply:
This would arise another discussion. A connector has the following gradle properties which lead to the connector name:

## Example 1: JetBrains Space

```
rootProject.name = "jetbrains-space"
connector.vendor=JetBrains
connector.productName=Space
```

The connector name in this case is actually set once to `JetBrains Space`

We could make it work with something like `"${connector.vendor} ${connector.productName}"`

## Example 2: Confluence Cloud

```
rootProject.name = "confluence-cloud"
connector.vendor=Atlassian
connector.productName=Confluence Cloud
```

How do we want to display confluence in the docs? As `Confluence Cloud` or `Atlassian Confluence Cloud`?

## Example 3: Clockify

```
rootProject.name = "clockify-connector"
connector.vendor=clockify
connector.productName=clockify
```

## Example 4: CSV

Does the csv connector even have a vendor?
This would result into another change in the connector registry.

This case would be trivial; if the vendor is equal to the product name just take one of it.

But in the end this leaves the example 2 open. Which one do you prefer?
I agree that this can be solved with a gradle task to remove this propert.

Reply:
The connector could simply be named `<vendor>-<productName>-connector`. And If `<vendor>` equals `<productName>` only use one.